### ISSUE TITLE
ExaMiniMD compile error

### ISSUE THREAD

**Issue by stanmoore1** [kokkos/kokkos#3117]
ExaMiniMD is failing to compile with the latest Kokkos develop: `make: *** No rule to make target 'Kokkos_MemorySpace.o', needed by 'libkokkos.a'. Stop.` Any ideas?

**Comment by stanmoore1** [comment]
ExaMiniMD built fine with 22b2cadeaf7d23139095cbb91a130baaaf53d3db and failed with 56777a79f1a2e4756fe64caf41dd8ea32d833697. Nothing changed in ExaMiniMD.

**Comment by masterleinad** [comment]
Can you check if
```diff
diff --git a/Makefile.targets b/Makefile.targets
index 5d738fbb..1d5b9866 100644
--- a/Makefile.targets
+++ b/Makefile.targets
@@ -32,6 +32,8 @@ Kokkos_SharedAlloc.o: $(KOKKOS_CPP_DEPENDS) $(KOKKOS_PATH)/core/src/impl/Kokkos_
        $(CXX) $(KOKKOS_CPPFLAGS) $(KOKKOS_CXXFLAGS) $(CXXFLAGS) -c $(KOKKOS_PATH)/core/src/impl/Kokkos_SharedAlloc.cpp
 Kokkos_MemoryPool.o: $(KOKKOS_CPP_DEPENDS) $(KOKKOS_PATH)/core/src/impl/Kokkos_MemoryPool.cpp
        $(CXX) $(KOKKOS_CPPFLAGS) $(KOKKOS_CXXFLAGS) $(CXXFLAGS) -c $(KOKKOS_PATH)/core/src/impl/Kokkos_MemoryPool.cpp
+Kokkos_MemorySpace.o: $(KOKKOS_CPP_DEPENDS) $(KOKKOS_PATH)/core/src/impl/Kokkos_MemorySpace.cpp
+       $(CXX) $(KOKKOS_CPPFLAGS) $(KOKKOS_CXXFLAGS) $(CXXFLAGS) -c $(KOKKOS_PATH)/core/src/impl/Kokkos_MemorySpace.cpp
 Kokkos_HostSpace_deepcopy.o: $(KOKKOS_CPP_DEPENDS) $(KOKKOS_PATH)/core/src/impl/Kokkos_HostSpace_deepcopy.cpp 
        $(CXX) $(KOKKOS_CPPFLAGS) $(KOKKOS_CXXFLAGS) $(CXXFLAGS) -c $(KOKKOS_PATH)/core/src/impl/Kokkos_HostSpace_deepcopy.cpp
 ```
works for you?

**Comment by stanmoore1** [comment]
Fixed by #3119.

**Comment by stanmoore1** [comment]
@masterleinad I didn't see your reply. This is now fixed.
